.env.example

```
@@ -99,6 +99,13 @@ BACKEND_CACHE_TTL_SUMMARY=3600
 BACKEND_LOG_LEVEL=INFO
 BACKEND_LOG_FILE=logs/dashboard.log
 
+# Monitoring / Metrics
+# Use the same token in the app stack and the monitoring stack
+BACKEND_METRICS_TOKEN=__GENERATE_RANDOM_TOKEN__
+MONITORING_METRICS_TOKEN=__SAME_AS_BACKEND_METRICS_TOKEN__
+GRAFANA_ADMIN_USER=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_USER key should go before the MONITORING_METRICS_TOKEN key

(UnorderedKey)